_projects/2022/100385586/100385586.Rmd

```diff
@@ -0,0 +1,310 @@
+---
+title: "Final project Alejandro Aísa" 
+description: | 
+  A replication of a graph from Pew Research Center on the 
+  restrictions on religion among the 25 most populous countries.
+categories: "2022"
+author: Alejandro Aísa 
+date: "`r Sys.Date()`"
+output:
+ distill::distill_article:
+   self_contained: false
+   toc: true 
+
+---
+
+## The original graph 
+
+The [graph to replicate](https://www.pewresearch.org/religion/interactives/restrictions-on-religion-among-the-25-most-populous-countries-2007-2020/) was originally created by the Pew Research Center. 
+It is an interactive graph depicting the evolution in time of two measures related to religious conflicts: _Goverment Restrictions Index_ (GRI) and _Social Hostilities Index_ (SHI). The 25 most populous countries are included in the graph. Our goal: to replicate it as closely as possible. 
+
+
+## Set up and libraries 
+
+As with any other project, we have to load some the libraries that will be needed to complete the graph.
+
+```{r setup, fig.showtext=TRUE, echo=TRUE, message=FALSE, warning=FALSE}
+library(tidyverse)
+library(gganimate)
+library(gifski)
+library(plotly)
+library(ggtext)
+library(RColorBrewer)
+library(htmlwidgets)
+library(ggtext)
+library(showtext)
+
+```
+
+## Prerequisites for the graph
+
+### Adapting the database. 
+
+First, we have to load the data. In this case, it has been directly compiled from the Pew Research Center. 
+
+Some columns are going to be renamed and/or modified to make easier the coding. Also, a new variable is created to give later the colour to the bubbles (geom_point()) via the case_when() function. 
+
+```{r}
+religion.df <- read_delim("rel.csv", show_col_types = FALSE)
+
+religion <- religion.df %>% transmute(
+  year = as.integer(`Year `), 
+  GRI = GRI, 
+  SHI = SHI,
+  Country = Country, 
+  Pop = `Population Size `) %>% 
+  mutate(
+  pos = case_when( # The position is numerically defined according to the score in the variables
```

Comment:
Please put this comment in its own line between the mutate and the case_when, so that it doesn't take so much width.